### PR TITLE
GUA-648: enabled International Phone Numbers feature upto staging env

### PR DESCRIPTION
**Pull request by TaherKapasi**
## Proposed changes
Enabled International Phone Numbers feature upto staging env

### What changed

SAM and Terraform config 

### Why did it change

Feature was disabled

### Issue tracking
- [GUA-648](https://govukverify.atlassian.net/browse/GUA-648)
- [GUA-625](https://govukverify.atlassian.net/browse/GUA-625)

### Environment variables or secrets
- `SUPPORT_INTERNATIONAL_NUMBERS` set to `1` upto staging env - still disabled in integration and production envs.

### Other considerations
- [ ] Demo to a BA, TA, and the team.
- [ ] Recorded demo shared on [#govuk-accounts-tech](https://gds.slack.com/archives/C011Y5SAY3U)
- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[GUA-648]: https://govukverify.atlassian.net/browse/GUA-648?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GUA-625]: https://govukverify.atlassian.net/browse/GUA-625?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ci/tasks/deploy-account-management.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ params:
   LOGGING_ENDPOINT_ARN: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prod"
   LOGGING_ENDPOINT_ENABLED: "true"
   DEPLOY_LISTENER: "true"
-  SUPPORT_INTERNATIONAL_NUMBERS: "0"
+  SUPPORT_INTERNATIONAL_NUMBERS: "1"
```
Comment (JHjava) [inline ci/tasks/deploy-account-management.yml:23]: This file was used whilst we deployed this app via concourse and could be removed now.

Comment (TaherKapasi) [reply ci/tasks/deploy-account-management.yml:23]: Hi @JHjava - we have some [tickets ](https://govukverify.atlassian.net/browse/ATB-203)to do this as part of migration that we will do once the migration and renaming is done - until then could we leave it in to be consistent.

Comment (JHjava) [reply ci/tasks/deploy-account-management.yml:23]: Ok cool

**Review by JHjava — APPROVED** [review]

**Review by domantas-gds — APPROVED** [review]
Approved - bit of tech debt down the line to deprecate the AMF terraform?

**Review by divyaparameswaran — APPROVED** [review]

**Review by huwd — APPROVED** [review]

**Pull request merged**